pkg/lint/rules/chartfile.go

```diff
@@ -22,11 +22,12 @@ import (
 	"os"
 	"path/filepath"
 
-	"github.com/Masterminds/semver/v3"
 	"github.com/asaskevich/govalidator"
 	"github.com/pkg/errors"
 	"sigs.k8s.io/yaml"
 
+	"github.com/Masterminds/semver/v3"
```

Comment:
What's with the move here? Can you move this back?

Reply:
It was moved by the `gitimports` command. I reverted the changes.